### ISSUE TITLE
trainning myself datasets 

### ISSUE THREAD

**Issue by IvyYZ** [kevinlin311tw/caffe-cvprw15#35]
My data set contains 10,200 images, and the next four images are classified as a single category . 
The number of my category is 2500. 
With the deephash network training, loss dropped from 7.1 to 3.79. Then iterate 50000 times, loss keeps stable.
I  want to know if this is a normal phenomenon. How can we let loss continue to decline?

**Comment by kevinlin311tw** [comment]
It depends on datasets. If you switch to other dataset, it is very possible that you have to customize the learning rates, and the iteration numbers (or epochs).

**Comment by koalaofpoint** [comment]
can you share the related dataset that you had got with me?,please @aaaaxiaodui

**Comment by aaaaxiaodui** [comment]
I only have some common datasets, I can send to you if you need!

**Comment by IvyYZ** [comment]
I have some common datasets, I can send to you if you need. @koalaofpoint
